### DISCUSSION TITLE
Blog post: Creating custom easing effects in CSS animations using the linear() function

### DISCUSSION THREAD

**Discussion by mauriciabad** [mdn/mdn-community#630] [Content]
About this post: https://developer.mozilla.org/en-US/blog/custom-easing-in-css-with-linear/

**Comment by mauriciabad** [comment]
In the **Resources** section, this tool could be added: (I made it)

- Easy Easings: https://easyeasings.mauri.app/

It lets users create `linear()` easings in a nice UI.

<img width="1728" alt="Screenshot 2024-01-19 at 1 34 32 PM" src="https://github.com/mdn/mdn-community/assets/12821361/52306ee9-f453-4e67-9472-3a6e6533ffda">

**Reply by bsmth** [reply]
That's nice, thanks a lot for sharing!
